clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -199,6 +199,35 @@ public void onMetadataUpdate(Cluster cluster) {
             new HashSet<>(Arrays.asList("topic", "topic1")), topics);
     }
 
+    @Test
+    public void testTopicExpiry() throws Exception {
+        // Test that topic is expired if not used within a refresh interval
+        long time = 0;
```

Comment:
That is good point. I think 0 is not a good sentinel value anyway since time is set to zero in so many tests. Hence changed the sentinel values to negative numbers.